glean-core/src/metrics/experiment.rs

```diff
@@ -101,23 +117,31 @@ impl ExperimentMetric {
         }
 
         // Make sure that branch id is within the expected limit.
-        let truncated_branch = if branch.len() > MAX_EXPERIMENTS_IDS_LEN {
+        let truncated_branch = if branch.len() > MAX_EXPERIMENTS_BRANCH_LEN {
```

Comment:
Let's keep using `MAX_EXPERIMENTS_IDS_LEN` here. This PR is specifically only about the `extra` :-)